src/store/RelayRecordStore.js

```diff
@@ -112,10 +113,11 @@ class RelayRecordStore {
   /**
    * Get the data ID associated with a storage key (and optionally an
    * identifying argument value) for a root query.
+   * toDO ?any should be something like number|string|object
    */
   getDataID(
     storageKey: string,
-    identifyingArgValue: ?string
+    identifyingArgValue: any
```

Comment:
ok, i modify it to mixed, i check that type inside function.   steveluscher told me to keep `identifyingArgValue` as a plain return value for `root node`.